app/backends/service-metrics.ts

```diff
@@ -4,7 +4,7 @@ import { getQueuedOrRunningCountForService } from '../models/user-work';
 import db from '../util/db';
 import logger from '../util/log';
 import { RequestValidationError } from '../util/errors';
-import { sanitizeImage } from '../util/string';
+import { sanitizeImage } from '@harmony/util/string';
```

Comment:
We purposely avoided this in the past - will relative paths no longer work?

Reply:
This is not the same as path aliases that Patrick didn't want to use before. The '@' symbol here is not referring to a path alias, it is referring to a namespace. This is using a namespace (@harmony) to group together libraries from a provider (in this case `harmony`). So we don't have to give them names like `harmony_env`, `harmony_string`, etc. This is just a normal library import now - so no, we won't use relative imports for the libraries we create as separate modules now. There is an entry for it in `package.json` `dependencies` like any other library, only it uses a local file path with no version string.